archean/egg-archean.json

```diff
@@ -0,0 +1,175 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-10-21T00:55:52+00:00",
+    "name": "Archean",
+    "author": "tueem@tomatentum.net",
+    "uuid": "01555bd3-155f-4ab6-b63d-c8bed0a3585f",
+    "description": "Archean invites you to unleash your creativity alone or with friends in a creative sandbox mode.Design and build vehicles, bases, rockets, space stations and much more using customizable modular blocks.Create unique configurations using components, program their behavior and explore an environment free of constraints.\\nTest your builds, refine your designs and defy the laws of physics.Archean is a game in development focused on engineering and build.",
+    "features": [],
+    "docker_images": {
+        "Ubuntu": "ghcr.io\/pelican-eggs\/yolks:ubuntu"
```

Comment:
Is ubuntu needed ? as most of our eggs use debian based images

Reply:
> Is ubuntu needed ? as most of our eggs use debian based images

no, its not. as i know, you can download the server via steam. so steam image would be better

Reply:
Looking at the docker images the files are on github

![afbeelding](https://github.com/user-attachments/assets/ef412e09-39d6-4779-bb9e-4617d1ec06d0)

https://github.com/batcholi/Archean_game_linux

Reply:
Is cloning a git repository acceptable? I figured that would be a less elegant solution than just using the docker image

Reply:
yes

Reply:
> Is ubuntu needed ? as most of our eggs use debian based images

Yes it is I tried debian first, but it has an outdated version of the libstdc++6 package whereby the server doesn't start

Reply:
Then ubuntu is fine.

Reply:
> Is cloning a git repository acceptable? I figured that would be a less elegant solution than just using the docker image

I would go with whatever is more reliable to be most up to date. Sometimes the building of the Docker image can be days behind what is currently live in the GitHub repo or SteamCMD, depending on how they have their CI/CD configured or if the image requires manual commits before it can be built. Therefore, it might be better to clone the repository, but I'll leave that choice up to you.